src/st/cen/D600.c

```diff
@@ -751,7 +751,30 @@ void EntityEnemyBlood(Entity* self) {
     }
 }
 
-INCLUDE_ASM("asm/us/st/cen/nonmatchings/D600", EntityUnkId08);
+extern u16 D_80180458;
+extern ObjInit2 D_8018125C[];
+void EntityUnkId08(Entity* entity) {
+    ObjInit2* objInit;
+
+    objInit = &D_8018125C[entity->subId];
```

Comment:
You can inline these :P
```suggestion
ObjInit2* objInit = &D_8018125C[entity->subId];